src/utilities/Fetch.js

```diff
@@ -1,14 +1,27 @@
 import * as API from './API'
```

Comment:
Anyone feel free to take out my comments if it seems we are all on the same page.